dbt/include/athena/macros/adapters/schema.sql

```diff
@@ -2,6 +2,9 @@
   {%- call statement('create_schema') -%}
     create schema if not exists {{ relation.without_identifier().render_hive() }}
   {% endcall %}
+
+  {{ adapter.add_lf_tags_to_database(relation) }}
+
 {% endmacro %}
 
```

Comment:
does dbt have an on update schema macro?

Reply:
@nicor88, I was thinking about the approach when we have database tags config on model level. But this is not good idea, as different models can have different configs for same database. That's why I've felt back to default tags for database on dbt profile level.

Reply:
ah good point - yeah, then better this way, tags for specific models can be overwritten per model basis - so all good.